plasmapy/formulary/tests/test_collisions.py

```diff
@@ -2102,6 +2116,50 @@ def test_limit_values(
                 calculated_limit_value, expected_limit_value, rtol=0.05, atol=0
             )
 
+    @pytest.mark.parametrize(
+        "frequency_to_test, constructor_keyword_arguments",
+        [
+            (
+                "Maxwellian_avg_ei_collision_freq",
+                {
+                    "test_particle": Particle("e-"),
```

Comment:
Add a test with Z>1 ions

Reply:
The formulary specifies that the limiting cases are for Z=1, is there some factor of Z I can multiply the expression by to get an expression for species Z > 1?

Reply:
Oh oops forgot about that, yep ok.